latex-syntax/src/main/java/org/xwiki/contrib/latex/internal/UIExtensionSupplier.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.contrib.latex.internal;
+
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Provider;
+import javax.inject.Singleton;
+
+import org.slf4j.Logger;
+import org.xwiki.component.annotation.Component;
+import org.xwiki.component.manager.ComponentLookupException;
+import org.xwiki.component.manager.ComponentManager;
+import org.xwiki.script.service.ScriptService;
+import org.xwiki.uiextension.UIExtension;
+import org.xwiki.uiextension.UIExtensionManager;
+import org.xwiki.uiextension.script.UIExtensionScriptService;
+
+/**
+ * Supply {@link UIExtension}s from user interface extension point identifiers. This component duplicates the logic of
+ * {@link UIExtensionScriptService#getExtensions(String)} outside of a {@link ScriptService}.
```

Comment:
would be good to link to a jira issue and put a TODO to remove it once that jira issue is fixed.

Reply:
I did it here https://github.com/xwiki-contrib/latex/pull/2/files#diff-4ffba4c8a4a0fb3db780aa2930fd3ddba2ba32ad68fe2c201f95bdaf1c2326b8R43
But it's probably not the right place indeed.

Reply:
ok my bad, I had missed it. I guess I was expecting it in the javadoc.